devtools/client/BUILD

```diff
@@ -11,8 +11,18 @@ vitest_config(name = "vitest_config")
 js_pipeline(
     package_name = "@player-tools/devtools-client",
     deps = [
-        ":node_modules/@player-tools/devtools-common",
-        "//:node_modules/@reduxjs/toolkit",
-        "//:node_modules/redux",
+        ":node_modules/@player-tools/devtools-messenger",
+        ":node_modules/@player-tools/devtools-types",
+        "//:node_modules/@chakra-ui/react",
+        "//:node_modules/@player-ui/pubsub-plugin",
+        "//:node_modules/@player-ui/react",
+        "//:node_modules/@devtools-ui/plugin",
```

Comment:
Does our devtool plugin need to be integrated here or should it be done at the actual extension level?